src/openshift/openshiftItem.ts

```diff
@@ -130,7 +141,7 @@ export abstract class OpenShiftItem {
         if (!context) context = await window.showQuickPick(OpenShiftItem.getProjectNames(), {placeHolder: projectPlaceholder});
         if (context && context.contextValue === ContextType.PROJECT && appPlaceholder ) {
             project = context as OpenShiftObject;
-            context = await window.showQuickPick<OpenShiftObject | QuickPickCommand>(OpenShiftItem.getApplicationNames(project, appPlaceholder && compPlaceholder === undefined), {placeHolder: appPlaceholder});
+            context = await window.showQuickPick<OpenShiftObject | QuickPickCommand>(OpenShiftItem.getApplicationNames(project), {placeHolder: appPlaceholder});
```

Comment:
It is way to many changes for fixing this issue. What if you would just change `appPlaceholder && compPlaceholder === undefined` to `appPlaceholder.includes('delete')`, would that work?